async-openai/src/client.rs

```diff
@@ -339,6 +339,30 @@ impl<C: Config> Client<C> {
         stream(event_source).await
     }
 
+    /// Make HTTP POST request to receive raw bytes for a stream based endpoint
+    pub(crate) async fn post_raw_stream<I>(
+        &self,
+        path: &str,
+        request: I,
+    ) -> Result<Pin<Box<dyn Stream<Item = Result<Bytes, OpenAIError>> + Send>>, OpenAIError>
+    where
+        I: Serialize,
+    {
+        let response = self
+            .http_client
+            .post(self.config.url(path))
+            .query(&self.config.query())
+            .headers(self.config.headers())
+            .json(&request)
+            .send()
+            .await
+            .map_err(OpenAIError::Reqwest)?;
+
+        let stream = response.bytes_stream().map(|item| item.map_err(OpenAIError::Reqwest));
```

Comment:
My guess is that its working without `stream=True` because this converts the actual non-streaming response into byte stream?

Reply:
See previous comment - from my tests, `stream=True` does not do anything... The response is streamed anyway from OpenAI side... If one has some doubts, one can confirm it by requesting a large text (> 2-3 paragraphs) and see the bytes being written to disk ~800ms  after the request.

Let me know if you cannot reproduce this